pkg/proc/stepping_test.go

```diff
@@ -956,7 +956,7 @@ func TestRangeOverFuncNext(t *testing.T) {
 					"result", "[]int len: 0, cap: 0, nil"),
 				nx(87), // if w == 2000
 				assertLocals(t, "result", "w"),
-				assertEval(t, "result", "[]int len: 1, cap: 1, [1000]"),
+				assertEval(t, "result", "[]int len: 1, cap: 2, [1000]"),
```

Comment:
If you changed this to make the linux/386 tests pass, the fix is different: see https://github.com/go-delve/delve/pull/3795